### ISSUE TITLE
[Question] Usage of test.slow() if beforeAll hook is slow

### ISSUE THREAD

**Issue by jithinjosejacob** [microsoft/playwright#14190]
Can we use test.slow() annotation in beforeAll hook  if before all execution is slow. or is it mandatorily needs to be on test blocks. I am trying to verify few errors in nth page. Where I need to fill out all pages until page n in beforeAll hook

**Comment by john-cyndx** [comment]
similarly, could there be a way to toggle or set the timeout countdown to begin only once the before hooks are done? I have tests where webkit is slow to load the page in the set up fixture and it means that my tests timeout randomly depending on the fixture load time. Could be a good feature

**Comment by dgozman** [comment]
@john-cyndx Do you use `beforeAll` or `beforeEach`? For `beforeAll`, you can set a separate timeout with `test.setTimeout()`. For `beforeEach`, you can increate test timeout if you'd like. Take a look [at the documentation](https://playwright.dev/docs/next/api/class-test#test-set-timeout).

**Comment by dgozman** [comment]
Sounds like everything is fixed here. Feel free to open a new issue and link to this one if there's still an issue.
